package/gluon-wan-dnsmasq/luasrc/lib/gluon/wan-dnsmasq/update.lua

```diff
@@ -36,13 +49,20 @@ end
 
 local static = uci:get_first('gluon-wan-dnsmasq', 'static', 'server')
 
+local handled_interfaces = handled_interfaces()
+
+if not util.contains(handled_interfaces, arg[1]) then
+  os.exit(0)
+end
```

Comment:
I'm not sure if this check is really useful - while checking inside instead of outside the script can avoid some redundancy, it is generally less efficient, and the current version always expects exactly one interface to be passed, which might not be sufficient for all usecases.

Reply:
From my POV this made a lot of sense if the behavior of `50-gluon-wan-dnsmasq` would be to call the script every time an interface comes up. But when we revert this, we can also drop this change.